client/src/App.js

```diff
@@ -1,17 +1,26 @@
+//Import is to bring information from the particular address into this file. This file is then able to export information.
 import React from "react";
```

Comment:
It might be helpful to note that this is actually pulling in the "react" module from our `node_modules` directory. If we don't have `react` added as a dependency in our package.json, there will be nothing to import